.env.example

```
@@ -16,10 +16,11 @@ STRIPE_SECRET_WEBHOOK_KEY=""
 DISCORD_BOT_TOKEN=""
 DISCORD_CLIENT_ID=""
 DISCORD_CLIENT_SECRET=""
-DISCORD_DAILY_ANIMAL_WEBHOOK_URL=""
-DISCORD_LEETCODE_DAILY_WEBHOOK_URL=""
-DISCORD_DAILY_REMINDERS_WEBHOOK_URL=""
-DISCORD_PRE_DAILY_REMINDERS_WEBHOOK_URL=""
+DISCORD_WEBHOOK_ANIMAL=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)